packages/filesystem/src/browser/location/location-renderer.tsx

```diff
@@ -61,12 +65,30 @@ class ResolvedDirectoryCache {
         });
     }
 }
+
+export const LocationListRendererFactory = Symbol('LocationListRendererFactory');
+export interface LocationListRendererFactory {
+    (options: LocationListRendererOptions): LocationListRenderer;
+}
+
+export const LocationListRendererOptions = Symbol('LocationListRendererOptions');
+export interface LocationListRendererOptions {
+    model: FileDialogModel;
+    host?: HTMLElement;
+}
+@injectable()
 export class LocationListRenderer extends ReactRenderer {
 
+    @inject(FileService) protected readonly fileService: FileService;
+    @inject(EnvVariablesServer) protected readonly variablesServier: EnvVariablesServer;
```

Comment:
minor: typo

```suggestion
    @inject(EnvVariablesServer) protected readonly variablesServer: EnvVariablesServer;
```

Reply:
Whoops, thanks for catching that. I've pushed the fix 👍